### ISSUE TITLE
CommandReturnCode for TemperatureMeasurement?  

### ISSUE THREAD

**Issue by gilhaz** [DanielOgorchock/OmniThing#10]
First, Your work is art! Thank you.

I wanted to use the `TemperatureMeasurement` device to monitor my RaspberryPi CPU temperature by running a script, something like this:

```
#!/bin/bash
CPU_TEMP=$(</sys/class/thermal/thermal_zone0/temp)
echo "$((CPU_TEMP/1000))"
```

But I don't see the `CommandReturnCode` as an input option, only `InputFloatRef`.

will `CommandReturnCode` be available in the future for `TemperatureMeasurement` device?

Thanks,
Gil :)

**Comment by DanielOgorchock** [comment]
I just released v0.7.0 which adds the CommandParser InputFloat type. It attempts to parse stdout of the command you give it for a float.

**Comment by gilhaz** [comment]
**Thank you** for your fast response! 
Works perfectly! 🥇
